google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java

```diff
@@ -46,8 +50,12 @@ private AssetSnippets() {}
   // [START securitycenter_list_all_assets]
   static ImmutableList<ListAssetsResult> listAssets(OrganizationName organizationName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // Start setting up a request for to search for all assets in an organization.
-      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
+      // Start setting up a request to search for all assets in an organization/ project/ folder.
```

Comment:
Global comment: Can we replace the slashes with commas?: "...in an organization, project, or folder."

This text appears in at least 7 places in the file.